Classes/CocoaLumberjack.swift

```diff
@@ -110,7 +109,7 @@ public func DDLogError(_ message: @autoclosure () -> String, level: DDLogLevel =
 /// Returns a String of the current filename, without full path or extension.
 ///
 /// Analogous to the C preprocessor macro `THIS_FILE`.
-public func CurrentFileName(_ fileName: StaticString = #file) -> String {
+public func currentFileName(_ fileName: StaticString = #file) -> String {
```

Comment:
We should still add a deprecated version of this, since this is otherwise a breaking change. I think there's comment to tell SwiftLint to ignore a line which we could add to the deprecated version.